src/services/rfqm_service.ts

```diff
@@ -699,6 +703,32 @@ export class RfqmService {
             ]);
         }
 
+        // verify that there is not a pending transaction for this taker and taker token
+        const pendingJobs = await this._dbUtils.findJobsWithStatusesAsync([
+            RfqmJobStatus.PendingEnqueued,
+            RfqmJobStatus.PendingProcessing,
+            RfqmJobStatus.PendingLastLookAccepted,
+            RfqmJobStatus.PendingSubmitted,
+        ]);
+
+        if (
+            pendingJobs.some(
+                (job) =>
+                    job.order?.order.taker === quote.order?.order.taker &&
+                    job.order?.order.takerToken === quote.order?.order.takerToken &&
+                    // Other logic handles the case where the same order is submitted twice
+                    job.metaTransactionHash !== quote.metaTransactionHash,
```

Comment:
Well one is coming from `rfqm_jobs` and the other is coming from `rfqm_quotes`. It probably won't happen but there was a test for it already and my new code broke the test because the error codes thrown were different.